MewsSystems/mews-flutter
# fix: Fix MultiselectSelectInput layout

#### Summary

- enabled SelectInput to expand in height if there are multiple selected values that would not fit.

#### Testing steps

1. Open Storybook Select Input story
2. Enable Multiselect
3. Select/deselect values and check if everything is working as intended.

#### Follow-up issues

None

#### Check during review

- Verify against Jira issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
